src/routers/secure/items.py

```
@@ -432,4 +433,71 @@ async def unblacklist_stream(_: Request, item_id: str, stream_id: int, db: Sessi
+                        media_item.store_state(None) # recheck the last state
+                        session.merge(media_item)
+                        session.commit()
+                        request.app.program.em.add_event(Event("RetryItem", media_item.id))
+                        logger.info(f"Successfully unpaused {media_item.log_string}")
+                    else:
+                        logger.debug(f"Skipping unpause for {media_item.log_string} - not in paused state")
+                except Exception as e:
+                    logger.error(f"Failed to unpause {media_item.log_string}: {str(e)}")
+                    continue
+    except ValueError as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)